classes/classes/Appearance.as

```diff
@@ -2228,7 +2147,7 @@
 					[SKIN_TYPE_GOO, "goo"],
 					[SKIN_TYPE_UNDEFINED, "undefined flesh"],
 					[SKIN_TYPE_DRAGON_SCALES, "scales"],
-					[SKIN_TYPE_FISH_SCALES, "scales"],
+					[SKIN_TYPE_FISH_SCALES, "scales"]
```

Comment:
Ah, I see. I saw that there was no comma on one of the Deer additions, so I wondered if that was what was supposed to be. Will adjust :-) Thank you~